src/Layout/SidebarLayout/SidebarLayout.component.tsx

```diff
@@ -93,29 +92,27 @@ export const SidebarLayout = ({
             contentWidth={contentWidth}
             contentBackgroundColor={contentBackgroundColor}
         >
-            {sidebarAlign === RIGHT ? (
-                <Grid columns={`1fr ${sidebarWidth}`}>
-                    <CustomAdaptor maxWidth={StandardBreakpoints.sm.max}>
-                        <Cell width={2}>{children}</Cell>
-                    </CustomAdaptor>
-
-                    <CustomAdaptor minWidth={StandardBreakpoints.md.min}>
-                        <Cell>{children}</Cell>
-                        <Cell>{sidebar}</Cell>
-                    </CustomAdaptor>
-                </Grid>
-            ) : (
-                <Grid columns={`${sidebarWidth} 1fr `}>
-                    <CustomAdaptor maxWidth={StandardBreakpoints.sm.max}>
-                        <Cell width={2}>{children}</Cell>
-                    </CustomAdaptor>
-
-                    <CustomAdaptor minWidth={StandardBreakpoints.md.min}>
-                        <Cell>{sidebar}</Cell>
-                        <Cell>{children}</Cell>
-                    </CustomAdaptor>
-                </Grid>
-            )}
+            {/*{sidebarAlign === RIGHT ? (*/}
```

Comment:
Preserving for when it's reimplemented?

Reply:
Yeah, I was going to just delete the entire layout directory until we decide what a non RMN layout implementation looks like, but I didn't want to cull this repo that hard when likely we'll want to bring it back soon.